### ISSUE TITLE
procedure with sysdate function not converted

### ISSUE THREAD

**Issue by pbidault** [darold/ora2pg#838]
Hello
I used the last ora2pg version
I tried to convert many procedures but ora2pg remains stuck on the only procedure using sysdate function
If I drop the procedure and I relaunch, all procedures are converted and ora2pg completes

source of the procedure ->
```
CREATE OR REPLACE PROCEDURE ALSEIS.AL_ALARM_INSERT
IS
--
BEGIN
--
DELETE ALSEIS.AL_ALARM
;
--
INSERT INTO
	ALSEIS.AL_ALARM
SELECT
	(SYSDATE - TO_DATE('01/01/1970','DD/MM/YYYY'))*24*60*60,
	NULL,
	SYSDATE
FROM
	DUAL
;
--
COMMIT
;
--
EXCEPTION
	WHEN OTHERS THEN
		RAISE
		;
--
END AL_ALARM_INSERT
;
/
```
ora2pg execution ->
```
siai-gatwv01:/home/tmp/ora2pg/AIAUTP_AEDBADMIN_ALSEIS # ora2pg -d -p -t PROCEDURE -o procedure.sql -b ./schema/procedures -c ./config/ora2pg.conf
WARNING: Enforcing KEEP_PKEY_NAMES to 1 as USE_TABLESPACE is enabled.
[2019-11-28 16:45:41] Ora2Pg version: 20.0
[2019-11-28 16:45:41] Trying to connect to database: dbi:Oracle:AIAUTP
[2019-11-28 16:45:41] Isolation level: SET TRANSACTION READ ONLY
[2019-11-28 16:45:41] Looking forward functions declaration in all schema.
[2019-11-28 16:45:43] Looking at package ujo_batch_pkg...
[2019-11-28 16:45:43] Looking at package ujo_chk_cond_pkg...
[2019-11-28 16:45:43] Looking at package ujo_det_rep_pkg...
[2019-11-28 16:45:43] Looking at package ujo_ext_batch_pkg...
[2019-11-28 16:45:43] Looking at package ujo_get_eoid_pkg...
[2019-11-28 16:45:43] Looking at package ujo_get_jobstart_depends_pkg...
[2019-11-28 16:45:43] Looking at package ujo_get_job_for_jil_pkg...
[2019-11-28 16:45:43] Looking at package ujo_get_job_pkg...
[2019-11-28 16:45:43] Looking at package ujo_get_legacy_job_pkg...
[2019-11-28 16:45:43] Looking at package ujo_get_machine_pkg...
[2019-11-28 16:45:43] Looking at package ujo_get_mach_jobs_pkg...
[2019-11-28 16:45:43] Looking at package ujo_get_preds_and_succs_pkg...
[2019-11-28 16:45:43] Looking at package ujo_get_queued_jobs_pkg...
[2019-11-28 16:45:43] Looking at package ujo_get_sched_info_pkg...
[2019-11-28 16:45:43] Looking at package ujo_put_job_resource_dep_pkg...
[2019-11-28 16:45:43] Looking at package ujo_setnget_pkg...
[2019-11-28 16:45:43] Looking at package AL_FORECAST_PACKAGE...
[2019-11-28 16:45:43] Looking at package AL_INVOICING_FORECAST_PACKAGE...
[2019-11-28 16:45:43] Looking at package AL_MASSP_FORECAST_PACKAGE...
[2019-11-28 16:45:44] Retrieving procedures information...
[2019-11-28 16:45:45] Add procedures definition...
[2019-11-28 16:45:45] Dumping function AEDBADMIN.UJO_WS_AUDIT_INFO_LOG...
[2019-11-28 16:45:45] Dumping to one file per function : AEDBADMIN.UJO_WS_AUDIT_INFO_LOG_procedure.sql
[2019-11-28 16:45:45] Parsing function aedbadmin.ujo_ws_audit_info_log...
[2019-11-28 16:45:45] Dumping function AEDBADMIN.UJO_WS_AUDIT_MSG_LOG...
[2019-11-28 16:45:45] Dumping to one file per function : AEDBADMIN.UJO_WS_AUDIT_MSG_LOG_procedure.sql
[2019-11-28 16:45:45] Parsing function aedbadmin.ujo_ws_audit_msg_log...
[2019-11-28 16:45:45] Dumping function AEDBADMIN.UJO_WS_DET_REP...
[2019-11-28 16:45:45] Dumping to one file per function : AEDBADMIN.UJO_WS_DET_REP_procedure.sql
[2019-11-28 16:45:45] Parsing function aedbadmin.ujo_ws_det_rep...
[2019-11-28 16:45:45] Dumping function AEDBADMIN.UJO_WS_GET_ID...
[2019-11-28 16:45:45] Dumping to one file per function : AEDBADMIN.UJO_WS_GET_ID_procedure.sql
[2019-11-28 16:45:45] Parsing function aedbadmin.ujo_ws_get_id...
[2019-11-28 16:45:45] Dumping function AEDBADMIN.UJO_WS_SEND_EVENT...
[2019-11-28 16:45:45] Dumping to one file per function : AEDBADMIN.UJO_WS_SEND_EVENT_procedure.sql
[2019-11-28 16:45:45] Parsing function aedbadmin.ujo_ws_send_event...
[2019-11-28 16:45:45] Dumping function ALSEIS.AL_ALARM_INSERT...
[2019-11-28 16:45:45] Dumping to one file per function : ALSEIS.AL_ALARM_INSERT_procedure.sql
[2019-11-28 16:45:45] Parsing function alseis.al_alarm_insert...
^CReceived terminating signal (INT).

siai-gatwv01:/home/tmp/ora2pg/AIAUTP_AEDBADMIN_ALSEIS/schema/procedures # ls -la
total 36
drwxr-xr-x  2 root root 4096 Nov 28 16:45 .
drwxr-xr-x 17 root root 4096 Nov 26 12:37 ..
-rw-r--r--  1 root root  807 Nov 28 16:45 AEDBADMIN.UJO_WS_AUDIT_INFO_LOG_procedure.sql
-rw-r--r--  1 root root  886 Nov 28 16:45 AEDBADMIN.UJO_WS_AUDIT_MSG_LOG_procedure.sql
-rw-r--r--  1 root root 3014 Nov 28 16:45 AEDBADMIN.UJO_WS_DET_REP_procedure.sql
-rw-r--r--  1 root root  667 Nov 28 16:45 AEDBADMIN.UJO_WS_GET_ID_procedure.sql
-rw-r--r--  1 root root 2180 Nov 28 16:45 AEDBADMIN.UJO_WS_SEND_EVENT_procedure.sql
-rw-r--r--  1 root root    0 Nov 28 16:45 ALSEIS.AL_ALARM_INSERT_procedure.sql
-rw-r--r--  1 root root  608 Nov 28 16:46 procedure.sql
-rw-r--r--  1 root root  632 Nov 28 16:45 temp_pass2_file.dat
```

Regards

**Comment by pbidault** [comment]
Hello
The problematic line is :
(SYSDATE - TO_DATE('01/01/1970','DD/MM/YYYY'))*24*60*60
ora2pg is able to convert ->
(SYSDATE - TO_DATE('01/01/1970','DD/MM/YYYY'))*24
but ora2pg is not able to convert and freezes ->
(SYSDATE - TO_DATE('01/01/1970','DD/MM/YYYY'))*24*60
Regards

**Comment by darold** [comment]
Fixed with commit 2d2e16d.
